openfasoc/generators/gdsfactory-gen/relational.py

```diff
@@ -0,0 +1,754 @@
+from pathlib import Path
+from typing import Optional, Union, Literal, Callable, cast
+from importlib import import_module
+import shutil
+import inspect
+import operator
+from abc import ABC, abstractmethod
+import nltk
+import re
+import copy
+
+
+def get_all_derived_classes(class_type, remove_parent: bool=True) -> list:
```

Comment:
Can you add docstrings to the leftover functions?

Reply:
done